### PR TITLE
ci: add integration with codeflow

### LINKED ISSUE #2945: 🤖 - codeflow integration
### Description

https://stackblitz.com/codeflow

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #2945

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +54,8 @@ use to design your app with Taiga UI components.
 
 ## Contributing
 
-See our [CONTRIBUTING.md](/CONTRIBUTING.md) guide
+See our [CONTRIBUTING.md](/CONTRIBUTING.md) guide. Try [make pull request](https://pr.new/github.com/tinkoff/taiga-ui)
```
Comment (waterplea) [inline README.md:57]: Either `making` or `to make`

File: projects/addon-doc/src/tokens/is-stackblitz.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import {inject, InjectionToken} from '@angular/core';
+import {LOCATION} from '@ng-web-apis/common';
+
+export const TUI_IS_STACKBLITZ_WEB_CONTAINER = new InjectionToken<boolean>(
+    `[TUI_IS_STACKBLITZ_WEB_CONTAINER]: Detect demo opened in Stackblitz webcontainer`,
+    {
+        factory: () => inject(LOCATION).hostname.endsWith(`webcontainer.io`),
```
[the remark this answers is not in the document]
Comment (waterplea) [reply projects/addon-doc/src/tokens/is-stackblitz.ts:7]: Maybe make it `.webcontainer.io`?

File: projects/addon-doc/src/tokens/base-href.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import {DOCUMENT} from '@angular/common';
+import {inject, InjectionToken} from '@angular/core';
+
+export const TUI_DOC_BASE_HREF = new InjectionToken<string | undefined>(
```
Comment (waterplea) [inline projects/addon-doc/src/tokens/base-href.ts:4]: We do we need it and not the Angular's `APP_BASE_HREF`?

**Review by waterplea — COMMENTED** [review]
Can you clarify base href logic, it's not clear to me.

File: README.md (hunk 1 above)
Comment (splincode) [reply README.md]: fixed

**Review by waterplea — APPROVED** [review]

**Comment by splincode** [comment]
@nsbarsukov @vladimirpotekhin please, review me

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**